javascript/ql/src/semmle/javascript/PackageExports.qll

```diff
@@ -29,33 +39,33 @@ PackageJSON getTopmostPackageJSON() {
 }
 
 /**
- * Gets a value exported by the main module from the package.json `packageJSON`.
+ * Gets a value exported by the main module from one of the topmost `package.json` files (see `getTopmostPackageJSON`).
  * The value is either directly the `module.exports` value, a nested property of `module.exports`, or a method on an exported class.
  */
-DataFlow::Node getAValueExportedBy(PackageJSON packageJSON) {
```

Comment:
This predicate removal is a breaking change.

Reply:
I know, but I had to change the API to fix performance for a repo. (It timed out otherwise).   
We got a `EXPERIMENTAL. This API may change in the future` comment in the top of the file, so I think it's OK.

Reply:
Agreed